package.json

```diff
@@ -24,11 +23,17 @@
     "typecheck": "turbo run typecheck --parallel",
     "clean": "turbo run clean --parallel && yarn clean:root",
     "clean:root": "rm -rf node_modules",
-    "knip": "knip --no-gitignore"
+    "clean:workspaces": "turbo run clean --parallel && yarn",
+    "knip": "knip --no-gitignore",
+    "check-references": "scripts/check-references.sh",
+    "fix-references": "scripts/fix-references.sh",
+    "generate-package": "plop --no-progress --plopfile 'templates/package/plopfile.mjs'",
+    "test-generated-package": "tsx scripts/testTemplatePackage.ts"
   },
   "devDependencies": {
     "@types/circomlibjs": "0.1.4",
     "@types/snarkjs": "0.7.5",
+    "plop": "^4.0.1",
```

Comment:
the names of the packages you've been adding frequently make we want to laugh. `knip` and `plop` ... lmao

Reply:
Plop is offensively bad. I'm sure fewer people use it because of the name. The scripts for it are called "plopfiles".